src/wp-admin/js/post.js

```diff
@@ -894,7 +1091,7 @@ jQuery(document).ready( function($) {
 
 			$document.off( '.wp-editor-resize' );
 
-			// sanity check
+			// Normalize height to stay within acceptable ranges.
```

Comment:
Maybe prefix with `Sanity check:` to make it clear that this is the sanity check the JSDoc for the function is talking about.